scripts/create_changelog.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
+set -u -e -o pipefail
 
 VERSION=$(grep '^VERSION' setup.py | sed -E "s/.*'([^']*)'/\1/")
 RELEASE_DATE=$(date "+%Y-%m-%d")
 
+# compile a list of already reported PRs
+tempfile="$(mktemp)"
+grep -E "#[0-9]{4}" -o CHANGELOG.md | sort > "$tempfile"
+
 echo "[$VERSION] - $RELEASE_DATE"
 echo "--------------------"
 echo '##### Enhancements'
 git log stable..master --first-parent --format='%s %b' |
 sed -E 's/.*#([0-9]+).*\[ENH\] *(.*)/\* \2 ([#\1](\.\.\/\.\.\/pull\/\1))/' |
-    grep -E '^\*'
+    grep -E '^\*' | grep -v -F -f "$tempfile"
```

Comment:
I don't see a reason for a new line, but if there are some bash style guidelines that someone prefers we can use them.

A better improvement would be to indent the line above.